test/tests.js

```diff
@@ -19,21 +19,24 @@ window.onload = function(){
 		ok( typeof( $.fn.snapper ) === "function" );
 	});
 
-	test( 'Enhancement steps', function() {
+	asyncTest( 'Enhancement steps', function() {
 		ok( $(".snapper_items > *:eq(2)").attr( "class" ) === undefined, "no classes applied at first" );
 		ok( $(".snapper_items > *:eq(2)").css( "float", "none" ), "no floats at first" );
 		ok( $(".snapper_items").attr( "style" ) === undefined, "no style inline applied" );
 		document.documentElement.className += "enhanced";
 		$(function(){
 			$( ".snapper" ).snapper();
+			start();
+			ok($(".snapper_items > *:eq(2)").css("float", "left"));
+			ok($(".snapper_items").attr("style") !== undefined, "style inline applied");
+			ok($(".snapper_items > *:eq(2)").attr("style") !== undefined, "style inline applied");
+			ok($(".snapper_pane").css("overflow-x") === "auto", "pane has overflow auto");
+			ok($(".snapper_pane").width() < $(".snapper_items").width(), "pane is narrower than items in it now");
+			ok($(".snapper_nextprev").length, "next prev generated");
+			ok($(".snapper_nextprev a").length === 2, "2 next prev links");
```

Comment:
@scottjehl 

Seems like many of these are tested elsewhere? For example `$(".snapper_nextprev").length` should be true for the other tests to pass as well. Not a big deal but it makes it a bit harder to debug test failures when more are failing than strictly necessary.

Reply:
yeah true. hmm. we could do a test cleanup - I moved these but didn't change them here.